internal/pkg/cli/push.go

```diff
@@ -65,11 +67,19 @@ func pushCommand(root *rootCommand) *cobra.Command {
 					return err
 				}
 
+				// Get encrypt plan
+				encryptPlan := plan.Encrypt(projectState)
```

Comment:
Myslim, ze sa da jednoducho zbavit duplicitneho kodu a zavolat cely `encrypt` prikaz:
```go
		Long:  pushLongDescription,
		RunE: func(cmd *cobra.Command, args []string) (err error) {
			// Run the encrypt command first, if the --encrypt flag is used
			if root.options.GetBool("encrypt") {
				encryptCmd := root.GetCommandByName("encrypt")
				if err := encryptCmd.RunE(encryptCmd, nil); err != nil {
					return err
				}
			}
			// ....

```

Reply:
Je tam len ten hacik, ze to musis dat hned na zaciatok `RunE`, ... ked to das do `action.action = func(`, tak by to neslo, pretoze:
- nacital by sa local state pre `push` command (nezasifrovane hodnoty)
- spustil by sa cely `encrypt` command, ... tj. znova by sa nacital local state, a ulozili by sa zasifrovane hodnoty
- vykonal by sa `push`, ... ale kedze sa stav nacital pred `encrypt`, tak by sa prepisali zasifrovane hodnoty nezasifrovanymi.

Reply:
ak to zavolam ako navrhujes tak sa vykona encrypt command s parametrami push commandu nie? tj dostane sa tam napr `--force` prepinac a potom encrypt command spadne na neznamom parametri, vyskusam to

Reply:
pojde to, ako som hovoril na calle, 

... este mi k tomu napadlo, pre lepsie vysvetlenie, ...v mojom priklade sa spusta priamo callback `encryptCmd.RunE`, teda nespusta sa cely prikaz "od zaciatku", prekoci sa parsovanie argumentov a ostatne veci, ktore tu nepotrebujeme.
A kedze `--dry-run` je uz nacitany z `push` cmd v `root.options` ... tak sa rovnako vyuzije aj v encrypt prikaze ... kedze `root.options` sa uz nemodifikuju.